sdk/servicebus/azure-servicebus/tests/test_mgmt_queues.py

```diff
@@ -0,0 +1,198 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import logging
+from typing import Type
+import pytest
+import uuid
+import datetime
+
+from azure.servicebus.management import ServiceBusManagementClient, QueueDescription
+from azure.servicebus._common.utils import utc_now
+from azure.core.exceptions import ResourceExistsError, HttpResponseError
+from devtools_testutils import AzureMgmtTestCase, CachedResourceGroupPreparer
+from servicebus_preparer import CachedServiceBusNamespacePreparer, ServiceBusQueuePreparer, CachedServiceBusQueuePreparer
+from utilities import get_logger
+
+_logger = get_logger(logging.DEBUG)
+
+class ServiceBusMgmtQueueTests(AzureMgmtTestCase):
+
+    @pytest.mark.liveTest
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    def test_mgmt_queue_create_by_name(self, servicebus_namespace_connection_string, **kwargs):
+        mgmt_service = ServiceBusManagementClient.from_connection_string(servicebus_namespace_connection_string)
+
+        queue_name = str(uuid.uuid4())
+        created_at = utc_now()
+        mgmt_service.create_queue(queue_name)
+
+        queue = mgmt_service.get_queue(queue_name)
+        assert queue.queue_name == queue_name
+        assert queue.entity_availability_status == 'Available'
+        assert queue.status == 'Available'
+        assert created_at < queue.created_at < utc_now() + datetime.timedelta(minutes=10) # TODO: Should be created_at_utc for consistency with dataplane.
```

Comment:
echoing my created_at_utc TODO comment, I believe we touched on briefly in conversation yesterday.

@annatisch screamtesting this one past you, my motivation was to normalize this with the other timestamp-esque things in the dataplane, but since this is mgmt libs I didn't know if I was colliding with any guidelines that would preempt.